sdcm/nemesis.py

```diff
@@ -344,7 +344,7 @@ def upgrade_node(self, node):
         node.remoter.run('sudo chown root.root /etc/yum.repos.d/scylla.repo')
         node.remoter.run('sudo chmod 644 /etc/yum.repos.d/scylla.repo')
         node.remoter.run('sudo yum clean all')
-        node.remoter.run('sudo yum update scylla scylla-conf scylla-server scylla-jmx scylla-tools -y')
+        node.remoter.run('sudo yum update scylla -y')
```

Comment:
My original cleanup is wrong, it doesn't work with latest package in centos.
recover it in latest patch.